src/index.html

```diff
@@ -10,7 +10,30 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body>      
+    <form class="search-bar" data-qa="big"> 
+      <input 
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search-bar__input search-bar__input--big"
+        data-qa="keypress"
+        >
+      <img 
+        src="images/Search.svg" 
+        alt="search icon" 
+        class="search-bar__icon--big"
```

Comment:
There is no needs to add modifier. Just use basic class search-bar__icon. 
```suggestion
        class="search-bar__icon"
```

Reply:
what if I use different styles for `search_bar__icon` small and big icons ? 
Because they have different position and sizes